src/App.css

```diff
@@ -0,0 +1,3 @@
+body {
+  display: block;
+}
```

Comment:
- [ ] you already styled your body in your index.css, so kindly remove this file to make your code base cleaner